helpdesk_mgmt/models/res_partner.py

```diff
@@ -23,14 +23,26 @@ class ResPartner(models.Model):
     )
 
     def _compute_helpdesk_ticket_count(self):
+        ticket_data = self.env["helpdesk.ticket"].read_group(
+            [("partner_id", "child_of", self.ids)], ["partner_id"], ["partner_id"]
+        )
+        active_ticket_data = self.env["helpdesk.ticket"].read_group(
+            [("partner_id", "child_of", self.ids), ("stage_id.closed", "=", False)],
+            ["partner_id"],
```

Comment:
Group instead by `commercial_partner_id`, and you will have all the results at once.

Reply:
In `helpdesk.ticket` there is no `commercial_partner_id` and you cannot group by `partner_id.commercial_partner_id` 
Do you mean something else?

Reply:
OK, my fault.